src/components/editor/EditorLogin.jsx

```diff
@@ -51,6 +85,29 @@ export default class Login extends BaseComponent {
     }
   }
 
+  onGoogleResponse(response) {
+    if (response.error) {
+      alert(`Google Login Failed.\n${response.error}`);
+      return;
+    }
+
+    const token = response.tokenId;
+
+    const xhr = new XMLHttpRequest();
+    xhr.open('POST', '/googlelogin', false);
+    xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+
+    xhr.onreadystatechange = () => {
```

Comment:
Nice, I for some reason mostly knew about the `addEventListener('load')` way, but after a bit of Googling this seems like the correct practice actually